Documentation/building/code-coverage.md

```diff
@@ -40,9 +40,9 @@ You can navigate to this from your PR by clicking the "Details" link to the righ
 
 ## Local Code Coverage Runs
 
-You can perform code coverage runs for the entire repository locally by adding the `coverage` argument to the `build -includetests` command.
+You can perform code coverage runs for the entire repository locally by adding the `coverage` switch (assuming that you  source and test assemblies are already built):
 
-    build -test -buildtests -coverage
+    build -test -coverage
```

Comment:
The comment right below says:

`This builds and tests the test assemblies and generates the full code coverage report. The resulting index.htm file providing the results of the run should be available at:`

Which is incorrect, since to build and test we need `-test -buildtests`

Reply:
thanks.